test/tools/auth0/handlers/branding.tests.js

```diff
@@ -19,22 +19,30 @@ describe('#branding handler', () => {
       });
     });
 
-    it('should update branding settings', async () => {
+    it('should update branding settings, removing template property', (done) => {
       const auth0 = {
         branding: {
           updateSettings: (params, data) => {
             expect(data).to.be.an('object');
+            expect(data.templates).to.equal(undefined);
             expect(data.logo_url).to.equal('https://example.com/logo.png');
-            return Promise.resolve(data);
+            done();
```

Comment:
The previous version of this test was fundamentally broken. The expectations were all in the stubbed `updateSettings()` function, this means that if that function call is completely removed from the production code, the test would run fine without any expectations ever being run at all.

To fix this, I've removed the `async` keyword, and instead added a `done()` callback to the stubbed function. This means that the test will fail (timeout) if the expected stub is not called.